misc/deployments/test4.gno.land/README.md

```diff
@@ -30,7 +30,7 @@ Some configuration params are required, while others are advised to be set.
 - `p2p.laddr` - the listen address for P2P traffic, **specific to every node deployment**. It is advised to use a
   reverse-proxy, and keep this value at `tcp://0.0.0.0:<port>`.
 - `p2p.max_num_outbound_peers` - the max number of outbound peer connections. **Advised to be `40`**.
-- `p2p.persistent_peers` - the persistent peers. ⚠️ **Required to be `g18vg9lgndagym626q8jsgv2peyjatscykde3xju@devx-sen-1.test4.gnodevx.network:26656,g1fnwswr6p5nqfvusglv7g2vy0tzwt5npwe7stvv@devx-sen-2.test4.gnodevx.network:26656,g1xa78fprcqcejfpk8xeycd4hzxtg56w9qe29xky@103.219.168.237:26656,g1h8dsnzlv7r4skfuud38runjuk4dnxenpr79meg@72.46.84.19:26656,g1ppdm4s90txrxu027j5et4crmxmmr3qr3g4wgrp@186.233.184.76:26656,g1hta5u3vmt4k2gklu5ashsl9q0my8ykzqu60vme@103.14.26.13:26656,g1tace0q5t06y3fhk2473uekl5hg3rphghdy6ykp@163.172.20.47:26656,g17958rreg27qmhq27tjrkuc9q4sjx9dchwywxk3@185.194.217.143:26656`** ⚠️.
+- `p2p.persistent_peers` - the persistent peers. ⚠️ **Required to be `g18vg9lgndagym626q8jsgv2peyjatscykde3xju@devx-sen-1.test4.gnodevx.network:26656,g1fnwswr6p5nqfvusglv7g2vy0tzwt5npwe7stvv@devx-sen-2.test4.gnodevx.network:26656,g1xa78fprcqcejfpk8xeycd4hzxtg56w9qe29xky@103.219.168.237:26656,g1h8dsnzlv7r4skfuud38runjuk4dnxenpr79meg@72.46.84.19:26656,g1ppdm4s90txrxu027j5et4crmxmmr3qr3g4wgrp@186.233.184.76:26656,g1hta5u3vmt4k2gklu5ashsl9q0my8ykzqu60vme@103.14.26.13:26656,g1tace0q5t06y3fhk2473uekl5hg3rphghdy6ykp@163.172.20.47:26656,g17958rreg27qmhq27tjrkuc9q4sjx9dchwywxk3@185.194.217.143:26656,g12p9l546ah4qeenhum8v4m2dg92jxcsrfy67yww@test4.gno.berty.io:26656`** ⚠️.
```

Comment:
i would recommend to use ip address over dns

Reply:
Ok I thought it was good because devx also uses DNS :)
In this case, we will wait a bit to set up the node in our final server (waiting after the provider...).
Thanks!

Reply:
We use indeed DNS but the DNS resolves always to a static IPs assigned when the network has been created.

If you look at the P2P module, before adding the peer to the list, the net go module resolves the DNS address to IP. As long as your IP is static, you can have your beauty sleep. 

Go ahead with DNS.

Reply:
Let me share also the rationale why it is better to have anyway static IP. The P2P tendermint module resolves DNS only at the bootstrap time of the node. if a remote peer for any reason change the IP, then the node will lose connection to it because internally it caches in the p2p persistent peers list the old IP.

Reply:
@D4ryl00 any updates on this?